### PR TITLE
Ft/remove forms

### LINKED ISSUE #16: Remove each form

### PR DESCRIPTION
**Pull request by Mohamilr**
You can delete entries now 

image

image

### CODE REVIEW

**Comment by adebayoileri** [comment]
Good, this works functionally well

**Comment by adebayoileri** [comment]
I'm thinking we could change the behaviour of the delete button appearing on all first sections i.e we should have at least one form section of the different parts of the CV  to be visible. 

 an additional check like this could do.


```
 setRecord({
                ...record,
                education: record.education.length > 1 && record.education.filter(  // d
                    (education, idx) => idx !== id
                ),
            });

```

File: pages/sections/CVForm.js [hunk 1]
```diff
@@ -441,6 +485,21 @@ export default function CVForm() {
                                 mt={"40px"}
                                 align="stretch"
                             >
+                                <Stack
```
Comment (adebayoileri) [inline pages/sections/CVForm.js]: or better still the delete button doesn't show when `index === 0`

**Review by adebayoileri — CHANGES_REQUESTED** [review]

File: pages/sections/CVForm.js (hunk 1 above)
Comment (Mohamilr) [reply pages/sections/CVForm.js]: alright, would update and revert

**Comment by Mohamilr** [comment]
Changes made

What do you think about the layout on the languages when the button is added

image

**Comment by adebayoileri** [comment]
> Changes made
> 
> What do you think about the layout on the languages when the button is added
> 
> image

Looks a bit odd, but since the language can be an optional field I guess we can add the delete button to the first item

**Comment by Mohamilr** [comment]
> > Changes made
> > What do you think about the layout on the languages when the button is added
> > image
> 
> Looks a bit odd, but since the language can be an optional field I guess we can add the delete button to the first item

Does it mean the language section will no longer appear when the inputs are deleted or will it still remain.

**Comment by adebayoileri** [comment]
> > > Changes made
> > > What do you think about the layout on the languages when the button is added
> > > image
> > 
> > 
> > Looks a bit odd, but since the language can be an optional field I guess we can add the delete button to the first item
> 
> Does it mean the language section will no longer appear when the inputs are deleted or will it still remain.

It can still appear would just show a placeholder e.g "No Available Languages"

**Comment by Mohamilr** [comment]
Done @adebayoileri 
image

**Review by adebayoileri — APPROVED** [review]

**Pull request merged**